tests/trace/test_client_trace.py

```diff
@@ -1508,25 +1506,39 @@ def op_with_unknown_types(
 
     assert len(inner_res.calls) == 1
     assert inner_res.calls[0].inputs == {
-        "a": repr(a),
+        "a": {
+            "__class__": {
+                "module": "test_client_trace",
+                "qualname": "test_unknown_input_and_output_types.<locals>.MyUnknownClassA",
```

Comment:
Why keep `<locals>`?

Reply:
I'm just storing the value exactly as it comes back from `obj.__class__.__qualname__` - but I'm not using the qualname for anything and I'm questioning whether it is worth saving or not in general.